### PR TITLE
feat(store): add materiel.net (FR) 

### PR DESCRIPTION
**Pull request by sirjeannot**
### Description
Hello,

pull request to add fr shop materiel.net.
some models were also added as all models listed on the shop were included, except watercooled card not AiO.
even through the shop is called 'materiel.net', I named it materielnet in case...

Thanks!

### Testing
Testing done on my end using docker, works fine. Reports correctly price, instock and not instock.

### New dependencies
None!

### CODE REVIEW

**Comment by sirjeannot** [comment]
to be honest, I have no clue why prettier isn't happy. I fixed all the other ones which were autotab from my editor.
```
Error:    7:17  error  Insert `⏎·······`  prettier/prettier
Error:   15:17  error  Insert `⏎·······`  prettier/prettier
```

File: src/store/model/store.ts [hunk 1]
```diff
@@ -135,14 +145,18 @@ export type Model =
   | 'pulse'
   | 'red devil'
   | 'red dragon'
-  | 'sg oc'
+  | 'rog strix'
+  | 'rog strix oc'
+  | 'rog strix oc white'
```
Comment (jef) [inline src/store/model/store.ts:150]:
```suggestion
```

Do you mind using the `strix oc`, `strix`, etc. We removed the `rog` as it's part of the brand.

**Review by jef — CHANGES_REQUESTED** [review]

File: src/store/model/store.ts (hunk 1 above)
Comment (sirjeannot) [reply src/store/model/store.ts]: the cleanup is done 👍
tested and ok.

**Comment by jef** [comment]
Hi @sirjeannot,

Sorry to say, but some of this code got superseded by https://github.com/jef/streetmerchant/pull/1836.

Are you able to resolve the conflicts? If so, I can pull it in afterward!

Thanks.

**Comment by sirjeannot** [comment]
done!

**Pull request closed without merging**